tests/automation_framework/src/utilities/worker_utilities.py

```diff
@@ -323,16 +367,21 @@ def retrieve_worker_id(pre_test_response):
             logger.error("ERROR: Failed to lookup worker")
     return worker_id
 
+
 def worker_retrieve_input(caller, input_json, pre_test_response):
```

Comment:
Anjali there are merge conflicts, and hence these characters have been left. will need fixing.